strings/tips/marine.txt

```diff
@@ -109,30 +107,27 @@ Chemrette cigarettes can be very useful.
 Sleeping will slowly heal cloneloss.
 As a mech-pilot, speed is not very important.
 As a mech-pilot, keep a close eye on your teamates. You lead the front!
-As a mech-pilot, the only weapon that have IFF is the sniper, everything else can friendly fire marines.
+As a mech-pilot, the only weapon that has IFF is the sniper, everything else can friendly-fire marines.
 As a mech-pilot, make sure to repair and reload often.
 As a flamer, you are a very valuable target. You clear resin walls and doors for marines to push. Just don't push marines into fire.
 Back fuel-tanks can be refilled via welding kits.
-The CSE can customize the detonation time of grenades with a screwdriver.
 White Phosphorus is very dangerous.
-Your gloves can be made into fingerless ones with wirecutters.
 Batteries for energy weapons can be primed into makeshift grenades with the help of a multitool.
 Gigachads leave the autodoc on automatic.
 You can fit tiny items into your helmet.
 The time it takes for you to wield your gun can be the difference between life and death.
 Switching to your sidearm is faster than reloading.
 You can preform a tactical reload by dragging a fresh magazine from your belt towards your gun.
-With your sidearm in hand, RMB a pistol pouch to instantly reload.
+With your sidearm in hand, click a pistol pouch to instantly reload.
```

Comment:
it specifically needs to be right mouse button. Otherwise you're just storing it.